### DISCUSSION TITLE
Dropdown doesn't show a muted placeholder

### DISCUSSION THREAD

**Discussion by alfanzain** [livewire/livewire#8371] [7. Bugs (archived)]
### Livewire version

v3.4.11

### Laravel version

v10.47.0

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

1. Copy code from https://livewire.laravel.com/docs/wire-model#select-dropdowns with some adjustment

```blade
<select wire:model="state">
    <option disabled>Select a state...</option>
 
    @foreach (\App\Models\State::all() as $state)
        <option value="{{ $state->id }}">{{ $state->label }}</option>
    @endforeach
</select>
```

2. Initialize `$state = ''` or `$state = null`

3.  The option selected for the first time is the second option, not the first option (the one that disabled).

### Repository or Wirebox URL

https://wirebox.app/b/yrnv2

### Additional information

Doc: https://livewire.laravel.com/docs/wire-model#select-dropdowns

I am not sure that the doc is wrong (beside of typos) or it's supposed to be like that.

I've create a Pull Request for updating the doc if it's wrong.
https://github.com/livewire/livewire/pull/8370

I've tried on two versions of Livewire in wirebox:

Livewire Version: v3.4.11
Laravel Version:  v10.47.0
PHP Version:      v8.1.12

https://wirebox.app/b/yrnv2

Livewire Version: v3.0.5
Laravel Version:  v10.24.0
PHP Version:      v8.1.12

https://wirebox.app/b/x21k0

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by dawiyo** [comment]
I’ve noticed the same issue. I don’t have a fix, but a workaround is to provide a BS value to the disabled option like this:

```blade
<option value="{{ microtime() }}" disabled>Select a state…</option>`

**Comment by alfanzain** [comment]
My PR https://github.com/livewire/livewire/pull/8370 was merged yesterday so I believe the docs was wrong.
